vendor/parser-extended/parser.mly

```diff
@@ -1493,32 +1494,30 @@ module_type_declaration:
 (* Opens. *)
 
 open_declaration:
-  OPEN
-  override = override_flag
-  ext = ext
-  attrs1 = attributes
-  me = module_expr
-  attrs2 = post_item_attributes
-  {
-    let attrs = attrs1 @ attrs2 in
-    let loc = make_loc $sloc in
-    let docs = symbol_docs $sloc in
-    Opn.mk me ~override ~attrs ~loc ~docs, ext
-  }
+    OPEN
+    override = override_flag
+    ext = ext
+    before = attributes
```

Comment:
I do not think you proposal is good idea, I know a lot of things are made like that in the ast, but i think labels shoulds descriptive in their context, and `before`/`after` is. I am willing to switch back to attrs1 and attrs2 and have `~before:attrs1`. This will also reduce the diff.

Reply:
I also just noticed that this PR uses the proposed convention a lot, so I am switching to it everywhere